app/src/test/java/org/fossasia/openevent/app/presenter/LoginPresenterTest.java

```diff
@@ -136,4 +144,19 @@ public void shouldNotAccessView() {
         Mockito.verifyNoMoreInteractions(loginView);
     }
 
+    @Test
+    public void shouldAttachEmailOnLoginSuccessfully() {
```

Comment:
Good work, also test that emails are attached automatically if present and not attached if they are not present. So, 2 more tests